script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_5_2.txt

```diff
@@ -1,13 +1,13 @@
 <Page1>
 I decide to stop her and ask for her name.
-I feel bad just calling her 'maid'.
+I feel guilty just calling her 'maid-san'.//　お手伝いさん、と呼びかけるのは申し訳ない気がしたからだ。
 <Page2>
 "Excuse me, may I ask your name?
-I just got here, so I don't really know what's going on."
+I just got here, so I haven't really grasped the situation here yet."//　来たばかりで、今のうちの事情がよく掴めなくて」
 I call out to her.
-The girl in the kimono looks at me in surprise.
+The girl in the kimono looks at me, eyes-wide.//　着物姿の少女は驚いたように目を見張った後、
```

Comment:
Does eyes wide have a hyphen?

Reply:
No lol. I thought for some reason the stanley kubrick movie had hyphens and that's why I did it but it turns out there's no hyphens there either